src/ui/Analytics/AnalyticsInformation.ts

```diff
@@ -45,17 +45,16 @@ export class AnalyticsInformation {
   }
 
   public clear() {
-    this.clearLocalStorage();
-    this.clearCookies();
+    this.clearVisitorId();
+    this.clearClientId();
   }
 
-  private clearLocalStorage() {
+  private clearVisitorId() {
     this.storage.removeItem(this.visitorIdKey);
-    new LocalStorageUtils(this.clientIdKey).remove();
   }
 
-  private clearCookies() {
-    Cookie.erase(this.visitorIdKey);
+  private clearClientId() {
+    new LocalStorageUtils(this.clientIdKey).remove();
     Cookie.erase(this.clientIdKey);
   }
```

Comment:
We never actually create a clientId, this is logic in case somebody has an old one.